README.md

```diff
@@ -25,3 +25,9 @@ Contributing to Flathub
 For information on creating packages or reporting issues please see the [contributing page](/CONTRIBUTING.md).
 
```

Comment:
This README shouldn't be part of your merge request, normally you would target the new-pr branch instead of master. Please rebase your merge request so that it doesn't contain history it doesn't belong to.